### ISSUE TITLE
animation options

### ISSUE THREAD

**Issue by jeremymanning** [ContextLab/hypertools#90]
we currently cater animations to data trajectories, but not point clouds.  this is because each animation frame reveals a new part of the data.  in other words, when `animate=True` we rotate the camera around the plot and produce a movie (default: `False`).

however, a different way of animating could work for point clouds as well: we could display the full dataset in each frame, but rotate the camera around the dataset visualization as in the current implementation.

for backwards compatibility, i propose we keep the current `animate` flag, but also add another animation-related flag, `evolve`.  when `evolve=True` (default), and if `animate=True`, then we produce an animation like the current one, where we introduce a new part of the data with each frame.  however, when `evolve=False` we display the entire dataset and rotate the camera around it.

in other words, regardless of the value of `evolve`, if `animate=True` we rotate the camera around the axes to produce a movie.  the `evolve` flag controls whether which data gets displayed in each frame changes frame-by-frame or not.
